server/config/machine-id.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"os"
+
+	"github.com/denisbrodbeck/machineid"
+)
+
+func GetMachineID() string {
+	id, err := machineid.ProtectedID("tracetest")
+	// fallback to hostname based machine id in case of error
+	if err != nil {
+		name, err := os.Hostname()
+		if err != nil {
+			return "default"
```

Comment:
In a worst-case scenario, it make sense to use a random string here?

Reply:
I rather have something that cannot be confused with a real id. So we can detect it